apps/test-viewer/src/components/quantity-formatting/FormatSetsTabPanel.tsx

```diff
@@ -0,0 +1,185 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import React from "react";
+import { Button, Flex, Text } from "@itwin/itwinui-react";
+import { FormatSetPanel, FormatSetSelector } from "@itwin/quantity-formatting-react";
+import { FormatManager } from "./FormatManager";
+
+import type { FormatSet } from "@itwin/ecschema-metadata";
+
+// Test format sets for development
+const testFormatSets: FormatSet[] = [
+  {
+    name: "TestFormatSet1",
+    label: "Arizona Highway Project (Civil)",
+    unitSystem: "imperial",
+    description:
+      "This format set contains all the formatting standards used by civil engineers on the Arizona Highway Project. Includes units for measurements, coordinates, and construction materials.",
+    formats: {},
+  },
+  {
+    name: "TestFormatSet2",
+    label: "Arizona Highway Project (Project Manager)",
+    unitSystem: "imperial",
+    description: "This format set contains all the formatting standards used by project managers on the Arizona Highway Project.",
+    formats: {},
+  },
+  {
+    name: "TestFormatSet3",
+    label: "My personal format set",
+    unitSystem: "metric",
+    description: "Custom formatting preferences for individual use. Combines metric and imperial units based on personal workflow requirements.",
+    formats: {},
+  },
+];
+
+/** Tab panel component for Format Sets management */
+export const FormatSetsTabPanel: React.FC = () => {
+  const [formatSets, setFormatSets] = React.useState<FormatSet[]>([]);
+  const [activeFormatSetKey, setActiveFormatSetKey] = React.useState<string | undefined>();
+  const [selectedFormatSetKey, setSelectedFormatSetKey] = React.useState<string | undefined>();
+  const [selectedFormatSet, setSelectedFormatSet] = React.useState<FormatSet | undefined>();
```

Comment:
Why do you need to track `selectedFormatSetKey` and `selectedFormatSet` separately? Wouldn't it be enough to track `selectedFormatSet`?

Reply:
yeah... removed